lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -315,17 +315,17 @@ class LLDBMemoryReader : public swift::remote::MemoryReader {
     if (name.empty())
       return swift::remote::RemoteAddress(nullptr);
 
-    LLDB_LOG(lldb_private::GetLogIfAllCategoriesSet(LIBLLDB_LOG_TYPES),
-             "[MemoryReader] asked to retrieve the address of symbol {0}",
-             name);
+    Log *log(GetLogIfAllCategoriesSet(LIBLLDB_LOG_TYPES));
+
+    LLDB_LOGV(log, "[MemoryReader] asked to retrieve the address of symbol {0}",
```

Comment:
Side note: I don't think we should print a prefix like `[MemoryReader] ` at all. There is a global flag that controls whether log messages should be prefixed with their function name, which I think serves the same purpose.

Reply:
Should I remove them before merging?

Reply:
If so, should we add some mention of "reflection"? Because something like "asked to retrieve the address of symbol" is a bit vague on details.

Reply:
This file has other tagged log messages. I'll do the untagging in a later change.

Reply:
You are right, the sentence also needs a subject. Perhaps "MemoryReader" instead of "[MemoryReader]"? Not important at any rate.